src/visitors/transpileCssProp.js

```diff
@@ -0,0 +1,108 @@
+// Most of this code was taken from @satya164's babel-plugin-css-prop
+// @see https://github.com/satya164/babel-plugin-css-prop
+import { useCssProp } from '../utils/options'
+
+const getName = (node, t) => {
+  if (typeof node.name === 'string') return node.name
+  if (t.isJSXMemberExpression(node)) {
+    return `${getName(node.object, t)}.${node.property.name}`
+  }
+  throw path.buildCodeFrameError(
```

Comment:
Oops! This function needs to accept `path` as an argument (or be defined inside the plugin closure to access path)

Reply:
Can you submit a PR? Thanks! :pray: (and let's hope nobody hits this anytime soon... :wink:)

Reply:
Will try to do it soon!